src/layout.css

```
@@ -520,3 +535,10 @@
     transform: translateY(12px);
   }
 }
+
+::view-transition-old(root),
+::view-transition-new(root) {
+  animation: none;
+
+  mix-blend-mode: normal;
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)